senderscore.py

```diff
@@ -3,25 +3,47 @@
 
 
 class SenderScore(BotPlugin):
-    """Plugin to show the mail score of the given ip"""
+    """Plugin to show the mail score of the given adress"""
 
     @botcmd
     def senderscore(self, msg, args):
```

Comment:
It ain't "args", why is it plural? why didn't you call it `hostname` like line 20